addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1028,7 +1028,7 @@ options.Class.include({
                 }
         }
 
-        if (params.reload || params.noBundleReload) {
+        if (params.reload || params.noBundleReload || widgetValue?.length === 0) {
```

Comment:
I dont think we need to check all those conditions above as well, right do we need to process the switch case if widgetValue is not there ?
We can simply return from the method from the first line.
what is your view on this